device/mellanox/x86_64-mlnx_msn2410-r0/plugins/psuutil.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
+
+#############################################################################
+# Mellanox
+#
+# Module contains an implementation of SONiC PSU Base API and
+# provides the PSUs status which are available in the platform
+#
+#############################################################################
+
+import os.path
+
+try:
+    from sonic_psu.psu_base import PsuBase
+except ImportError, e:
+    raise ImportError (str(e) + "- required module not found")
+
+class PsuUtil(PsuBase):
```

Comment:
>PsuUtil [](start = 6, length = 7)

Extract base class for mlnx?

Reply:
@qiluo-msft: I don't understand what you intend to accomplish here. Could you elaborate?

Reply:
(refine original comment)